.github/issue_template.md

```diff
@@ -0,0 +1,16 @@
+## Expected Behavior
+
+
+## Actual Behavior
+
+
+## Steps to Reproduce the Problem
+
+1.
+2.
+3.
+
+## Specifications
+
+- Version:
```

Comment:
hence `m.test` is quite platform dependent i guess it would be appropriate to list both node and npm version.